file-formats/edck/type/zif_aff_edck_v1.intf.abap

```diff
@@ -0,0 +1,72 @@
+INTERFACE zif_aff_edck_v1
+  PUBLIC.
+
+  TYPES ty_check_classification TYPE c LENGTH 1.
+  TYPES ty_check_category TYPE c LENGTH 6.
+
+  TYPES:
+    BEGIN OF ty_attributes,
+      "! <p class="shorttext">Check Category</p>
+      "! Consistency check category
+      "! $values {@link zif_aff_edck_v1.data:co_check_category}
+      "! $default {@link zif_aff_edck_v1.data:co_check_category.existence_check}
+      check_category              TYPE ty_check_category,
+      "! <p class="shorttext">Check Classification</p>
+      "! Consistency check ID classification
+      "! $values {@link zif_aff_edck_v1.data:co_check_classification}
+      "! $default {@link zif_aff_edck_v1.data:co_check_classification.core}
+      check_id_classification     TYPE ty_check_classification,
+      "! <p class="shorttext">Additional Check Class</p>
+      "! Consistency check class
+      check_class                 TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Result Derivation Indicator</p>
+      "! Result process derived by BAdI
+      result_derivation_indicator TYPE abap_bool,
```

Comment:
As already discussed, would something like `result_process_derived` be possible?